.chloggen/support-other-quantiles-of-cw-metrics.yaml

```diff
@@ -0,0 +1,27 @@
+# Use this changelog template to create an entry for release notes.
+
+# One of 'breaking', 'deprecation', 'new_component', 'enhancement', 'bug_fix'
+change_type: enhancement
+
+# The name of the component, or a single word describing the area of concern, (e.g. filelogreceiver)
+component: awsfirehosereceiver
+
+# A brief description of the change.  Surround your text with quotes ("") if it needs to start with a backtick (`).
+note: support all possible quantile values (0.7, 0.8, 0.9, 0.95, 0.99) of CloudWatch metrics in addition to quantile 0 (min) and 1 (max)
```

Comment:
IIANM you can specify arbitrary percentiles, and you're not restricted to the ones in the drop down in CloudWatch